cmd/mattermost/commands/export.go

```diff
@@ -154,6 +154,7 @@ func buildExportCmdF(format string) func(command *cobra.Command, args []string)
 
 		if a.MessageExport == nil {
 			CommandPrettyPrintln("MessageExport feature not available")
```

Comment:
We still need the `CommandPrettyPrintln`, is not already handled automatically by the `error`?

Reply:
I removed it, if you think it is superfluous, wasn't sure if we want to keep both, since one goes to stdout and one to stderr I presume.